src/content/en/fundamentals/media/manipulating/cheatsheet.md

```diff
@@ -191,11 +194,6 @@ remove all whitespace including the final carriage return.
 For the `-key` flag use the key created earlier and stored in the media.key
 file. However, when entering it on the command line, be sure you've removed its
 whitespace. For the `-key_id` flag repeat the key value.
```

Comment:
That's what I thought you meant. But I haven't created the MPD yet. Is there some place else I can get it?